### ISSUE TITLE
FEL mode not working on NES Mini

### ISSUE THREAD

**Issue by Nate3611** [ClusterM/hakchi2#588]
Any help would be great.  I'm trying to add games to my NES mini and when I try putting it into FEL mode the power led turns on and I know it is to stay off.  I've tried it being connected to different computers, different USB cables, different USB ports, installed the driver, and tried different versions of Hakchi2.  I follow the steps on Hakchi2 I get to the point where I press and hold the reset button and then press the power button and nothing happens on the computer and if I wait a few seconds or more and release the reset button after a few seconds the power led turns on.  On my computer it is showing up as a Media Player under USB.  The computers I have tried it on are running Windows 7 and Windows 10 and both are current on updates.  Is there something I'm doing wrong? Or has anyone else run into this problem?

**Comment by shaman78** [comment]
try release 2.17d
 2.20 is a bit buggy for nes

**Comment by Nate3611** [comment]
The issue is the nes system.  I did some digging and found out the one I have is a fake knock off that looks like the real one, but it didn't come from Nintendo so it can't be hacked.

**Comment by profundiscom** [comment]
How did you determine it was a knockoff? I have been having a similar issue with one I purchased off eBay and it looks fairly genuine.

The one I have will not go into FEL mode (light off). And if I do have a knockoff, can I solder or jumper someting inside to put it into FEL mode.. I mean, there's just gotta be a way to get more games on this thing.

**Comment by Nate3611** [comment]
I started looking around online and found some videos on YouTube.  I actually watched this one https://www.youtube.com/watch?v=o_WiF0KB2IE

and mine has all of the exact things that the fake one on this video has.  I looked around to see if it was possible to get more games on the fake ones and no one has done it yet and from what I found the fake ones use a different system that is why they do not go into the FEL mode.  I got mine off of Ebay as well and was not happy at all so I contacted the seller and they are refunding my money.  It was past the return date, but I told them what was going on and that I found out that it was a fake and would report them to Ebay if they didn't refund my money and they didn't give me any issues.
